### PR TITLE
Make more comprehensible the french documentation

### PR DESCRIPTION
**Pull request by ponsfrilus**
Still some improvement works to do, but it's a little bit better.

### CODE REVIEW

**Comment by dblotsky** [comment]
The changes seem fine, but they should be made in crowdin. The changes made in this PR would get overwritten by translation builds.

**Comment by stevengill** [comment]
You can find it at https://crowdin.com/project/cordova

File: www/docs/fr/dev/guide/overview/index.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ Apache Cordova a obtenu son diplôme en octobre 2012 comme un projet de niveau s
...
 
 *   un développeur Web et que vous souhaitez déployer une application Web prête à être distribuée dans divers portails de vente d'applications.
 
 *   un développeur de mobile intéressé par la combinaison de composants de l'application native avec une *WebView* (fenêtre de navigateur spécial) qui peut accéder aux API de niveau périphérique, ou si vous voulez développer une interface plugin entre les autochtones et les composants WebView.
 
 ## Composants de base
 
-Apache Cordova applications s'appuient sur un commun `config.xml` fichier qui fournit des informations sur l'application et spécifie les paramètres qui affectent comment ça marche, comme si elle répond à l'orientation se déplace. Ce fichier respecte [Emballés Web App][2]ou *widget*, spécification de la W3C.
+Les applications Apache Cordova s'appuient sur le fichier `config.xml` qui fournit des informations sur l'application et qui spécifie des paramètres qui affectent leurs fonctionnements, comme par exemple la rotation automatique de l'écran. Ce fichier respecte la [spécification de la W3C sur les widgets][2].
```
Comment (akofman) [inline www/docs/fr/dev/guide/overview/index.md]: "de la W3C" should be "du W3C"

**Comment by dblotsky** [comment]
Here's the direct Crowdin link: https://crowdin.com/translate/cordova/12/en-fr. You'll need to create a Crowdin account to edit it though.

**Comment by ponsfrilus** [comment]
@dblotsky @stevengill @akofman thanks guy. I already tried on crowdin before I found the github page. Crowdin seems to me to be ardous... Anyhow will give it another try when I get some time.

**Pull request closed without merging**